modules/unsupported/stac-store/src/main/java/org/geotools/stac/store/STACDataStoreFactory.java

```diff
@@ -0,0 +1,116 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2022, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.stac.store;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import org.geotools.data.DataStore;
+import org.geotools.data.DataStoreFactorySpi;
+import org.geotools.data.Parameter;
+import org.geotools.http.commons.MultithreadedHttpClient;
+import org.geotools.stac.client.STACClient;
+import org.geotools.stac.client.STACClient.SearchMode;
+import org.geotools.util.KVP;
+
+public class STACDataStoreFactory implements DataStoreFactorySpi {
+
+    public static final Param DBTYPE =
+            new Param(
+                    "dbtype",
+                    String.class,
+                    "Type",
+                    true,
+                    "stac",
+                    Collections.singletonMap(Parameter.LEVEL, "program"));
+
+    public static final Param LANDING_PAGE =
+            new Param("landingPage", URL.class, "The STAC server landing page URL", true);
+
+    public static final Param NAMESPACEP =
```

Comment:
is the final "P" intentional? NAMESPACEP? (is it for Param to avoid conflict with other NAMESPACE around?)

Reply:
I copied it from another factory but indeed... no need for the P.